src/main/java/com/suse/salt/netapi/calls/LocalCall.java

```diff
@@ -1,23 +1,24 @@
 package com.suse.salt.netapi.calls;
 
-import static com.suse.salt.netapi.utils.ClientUtils.parameterizedType;
-
+import com.google.gson.reflect.TypeToken;
 import com.suse.salt.netapi.AuthModule;
 import com.suse.salt.netapi.client.SaltClient;
+import com.suse.salt.netapi.datatypes.Batch;
 import com.suse.salt.netapi.datatypes.target.Target;
 import com.suse.salt.netapi.exception.SaltException;
 import com.suse.salt.netapi.results.Result;
 import com.suse.salt.netapi.results.Return;
 import com.suse.salt.netapi.results.SSHResult;
 
-import com.google.gson.reflect.TypeToken;
-
 import java.lang.reflect.Type;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
 import java.util.Optional;
 
+import static com.suse.salt.netapi.utils.ClientUtils.parameterizedType;
+
+
```

Comment:
Please don't (let your IDE) reorder imports randomly, same above with `TypeToken`. Your patch should contain only changes that are actually relevant.